webapp/src/login/AdminLoginPage.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import shinobiStackLogo from '../assets/shinobistack.png';
+
+const AdminLoginPage = () => {
+  const [adminSecret, setAdminSecret] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // TODO: Handle admin login logic here
+    console.log('Admin Secret:', adminSecret);
+  };
+
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-gray-300">
+      <div className="w-full max-w-md p-8 space-y-6 bg-white rounded-3xl shadow-md">
+        <img 
+            src={shinobiStackLogo}
+            alt="Admin Logo" 
+            className="w-full h-auto mb-4 rounded-3xl" 
+          />
+        <h2 className="text-2xl font-bold text-center">gokakshi</h2>
+        <h3 className="text-1xl text-center">The centralized security platform</h3>
```

Comment:
Just noticed spelling. 
It needs to be `gokakashi`

Reply:
yep, added a fix as part of https://github.com/shinobistack/gokakashi/pull/72 (which the next PR to get merged)